src/modules/watch/service.js

```diff
@@ -1,8 +1,8 @@
 const { createTimestampHook } = require('@albert-team/mongol/builtins/hooks')
-const axios = require('axios')
 const { ObjectID } = require('mongodb')
+const { Scheduler } = require('../../externals')
 
-const { GATEWAY_ADDRESS } = process.env
+const schedulerService = new Scheduler()
```

Comment:
Thực ra ý kiến hay đấy, t nghĩ có thể đổi từ `Scheduler` -> `SchedulerService` thì hợp lý hơn
```suggestion
const schedulerService = new SchedulerService()
```